docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -193,6 +193,47 @@ The POST request body should be in JSON format with the folowing structure:
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - signature verification or name address ownership error.
 
+### Updating name addresses
+
+Used to update or add name addresses.
+
+`POST /v1/profile/account/{name}/address`
+
+* `name` - is the registered name. eg. `coolname.wc.ink`
+
+#### Request body:
+
+The POST request body should be in JSON format with the folowing structure:
+
+```typescript
+{
+    "message": string,
+    "coin_type": string,
+    "signature": string,
+    "address":string,
```

Comment:
```suggestion
    message: string,
    coin_type: string,
    signature: string,
    address: string,
```

Reply:
I see that we have such a convention in [notify as well](https://github.com/WalletConnect/walletconnect-specs/blob/65b0deaf1738bfd53e8be0de98acbb7928d92560/docs/specs/servers/notify/notify-server-api.md?plain=1#L17-L22) 👀 so if we are going to change it, it's better to change everywhere in a followup.